src/main/main.ts

```diff
@@ -757,6 +757,7 @@ app.on("ready", () => {
         quitApp();
     }
 });
+app.setAppUserModelId("ueli.com");
```

Comment:
Can you explain why this is needed?

Reply:
the reason that i added this is line is because electron notification doesn't show otherwise in windows:
https://www.electronjs.org/docs/tutorial/notifications#windows

Reply:
this the notification with the line:
![image](https://user-images.githubusercontent.com/9766462/81399328-bd5b1500-9122-11ea-9cf2-5edbba4d1d49.png)
this is without:
![image](https://user-images.githubusercontent.com/9766462/81399462-f4c9c180-9122-11ea-949c-71a0272e3f19.png)
do you want to change it to something else or remove the line entirely?